### PR TITLE
The Threat detail view now display the library name instead of its id

### PR DESCRIPTION
**Pull request by monsieurswag**

### CODE REVIEW

File: backend/core/serializers.py [hunk 1]
```diff
@@ -174,6 +186,7 @@ class Meta:
 
 class ThreatReadSerializer(BaseModelSerializer):
     folder = FieldsRelatedField()
+    library = LibraryReadSerializer()
```
Comment (nas-tabchiche) [inline backend/core/serializers.py]: Serializing the entirety of each threat's library is overkill, as only their names and urns are needed.

**Review by nas-tabchiche — CHANGES_REQUESTED** [review]
Use `FieldsRelatedField` with the fields `name` and `urn` to avoid fetching unused data when fetching threats

**Review by eric-intuitem — APPROVED** [review]
LGTM

**Review by eric-intuitem — CHANGES_REQUESTED** [review]
Not working for reference controls

**Review by eric-intuitem — APPROVED** [review]
LGTM now

**Review by eric-intuitem — APPROVED** [review]
LGTM

**Comment by eric-intuitem** [comment]
Nassim's request has been addressed.

**Pull request merged**